### DISCUSSION TITLE
next-intl/navigation not exporting createSharedPathnamesNavigation (app router)

### DISCUSSION THREAD

**Discussion by duckycoding-dev** [amannn/next-intl#621] [General]
It seems like createSharedPathnamesNavigation doesn't get exported from the navigation module
![image](https://github.com/amannn/next-intl/assets/150606804/75510d06-f2a5-45c5-82b0-146a4ced2b22)
![image](https://github.com/amannn/next-intl/assets/150606804/3b26ac15-e1d2-4306-9619-4830d3e9093b)

Or am I missing something? I think the export was removed by accident, I would just add it back like this locally
![image](https://github.com/amannn/next-intl/assets/150606804/eb0e5d49-2b52-4f5f-a0d9-fc268c327a96)

**Comment by amannn** [comment]
Which version of `next-intl` are you using? Seems to be [working fine for me with the latest release candidate](https://codesandbox.io/p/sandbox/next-intl-bug-template-app-forked-6rgf3f?file=%2Fsrc%2Fapp%2F%5Blocale%5D%2Fpage.tsx%3A8%2C7-8%2C62) (`next-intl@3.0.0-rc.10`).

**Reply by duckycoding-dev** [reply]
Mhmmm, seems like I was using a beta version, 3.0.0-beta.19, maybe that's why
My bad 🙏
